client/src/components/BookDetail.js

```diff
@@ -0,0 +1,139 @@
+import PropTypes from 'prop-types'
+import RectangleButton from './RectangleButton'
+import styled from 'styled-components/macro'
+
+/** @component */
+export default BookDetail
+
+BookDetail.propTypes = {
+  detailBook: PropTypes.array,
+  onButtonClick: PropTypes.func,
+  buttonName: PropTypes.string,
+}
+
+function BookDetail({ detailBook, onButtonClick, buttonName }) {
+  return (
+    <BookDetailModal>
+      {detailBook.map((detailBook) => (
+        <StyledDetailWrapper key={detailBook.id}>
```

Comment:
Again the styled prefix, also `StyledCover`, `StyledDetails`. 
I think you can just leave out the "styled" completely.